dspace-api/src/main/java/org/dspace/handle/service/HandleClarinService.java

```diff
@@ -215,4 +215,14 @@ public void update(Context context, Handle handleObject, String newHandle,
      * @throws SQLException if database error
      */
     public String getDeadSince(Context context, String handle) throws SQLException;
+
+    /**
+     * Create handle without dspace object.
```

Comment:
If this is only for migration purposes please add some note that this is only for migration and for a specific case.